src/Vts.Scripting/ForwardSolvers/Demo05PHDOfRhoAndZ.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Vts.Common;
+using Vts.Modeling.ForwardSolvers;
+using Plotly.NET.CSharp;
+using Vts.Factories;
+
+namespace Vts.Scripting.ForwardSolvers;
+
+/// <summary>
+/// Class using the Vts.dll library to demonstrate predicting photon hitting density (PHD)
+/// as a function of radial extent and depth at a given set of optical properties and source-detector separation
+/// </summary>
+internal class Demo05PHDOfRhoAndZ : IDemoScript
```

Comment:
This name "should" be Demo05PhdOfRhoAndZ

Having said that I do realize that this is wrong in many places in our solution, my preference would be to change it here but I do understand if you want to keep it this way.

Reply:
It's kind of a question of "glanceable comprehension" by a photonics domain expert, vs correctness from a pure C# developer perspective. The function that generates it is, itself, `GetPHD` (which was intentional when we created it). Same thing with CAW vs DAW, etc.

Reply:
05PHDO is what I see when I glance. I'm not sure the answer to this one. I had a tough time with this in the beginning for things like PDF and API but now I abide by it because these are know acronyms and the Pascal case defines the separation. Honestly, here I would be ok with either way.

@hayakawa16 and I have been changing these in the code to follow C# standards. At the end of the day if you are programming in a language is is good to abide by the rules of the language. For example wouldn't this be in lower case delimited with underscores in Python?

Reply:
I see what you mean (one reason I had an underscore in there originally). Either is fine with me, "ideomatic C#" and "ideomatic Python" probably make sense. Yes, Python often uses underscores in names, sometimes `PragmaticCamelCase_WithAbbreviationsAndUnderscoresLike_RGB()` for class members.

Reply:
Here are all the changes I've applied, but haven't yet committed (all make sense?):

`Demo05PHDOfRhoAndZ` -> `Demo05PhdOfRhoAndZ`
`Demo07PHDOfRhoAndZTwoLayer` -> `Demo07PhdOfRhoAndZTwoLayer`
`Demo02DAWvsCAW` -> `Demo02DawVsCaw`
`Demo04N1000vsN100` -> `Demo04N1000VsN100`
`Demo06pMCPostProcessor` -> `Demo06PmcPostProcessor`
`Demo07pMCInversion` -> `Demo07PmcInversion`

Reply:
I like those, do you want to fix the OP -> Op also? :)

Reply:
No, it's a two letter acronym and not an abbreviation

https://stackoverflow.com/a/596070

Reply:
That post is 14 years old, Microsoft call it Id now and Db etc.

Maybe write out OpticalProperties then like we do in the code?

Reply:
> By convention, two-letter acronyms use all uppercase letters, and acronyms of three or more characters use Pascal casing. The following examples use this naming convention: 'DB', 'CR', 'Cpa', and 'Ecma'. The following examples violate the convention: 'Io', 'XML', and 'DoD', and for non-parameter names, 'xp' and 'cpl'. 'ID' is special-cased to cause a violation of this rule. 'Id' is not an acronym but is an abbreviation for 'identification'.

https://learn.microsoft.com/en-us/visualstudio/code-quality/ca1709?view=vs-2022

Reply:
It's also true that abbreviations are discouraged. If we think the extra name length isn't ridiculous, I'm ok spelling them out, just seemed long.

Reply:
That's a better article, makes sense.

Reply:
> It's also true that abbreviations are discouraged. If we think the extra name length isn't ridiculous, I'm ok spelling them out, just seemed long.

Your call, I like writing things out also but I can live with this if it's comprehendible.

Reply:
Slept on it - thinking "OpProp" is a good balance to strike

Reply:
That's funny, you had just convinced me that 2 letter acronyms were acceptable as uppercase, now I am ok with IO and DB. I do think these are more readable the way you have them though.